### DISCUSSION TITLE
Version for Mojave?

### DISCUSSION THREAD

**Discussion by MrPoetry** [mikekazakov/nimble-commander#492] [Q&A]
Is there a version that still runs on Mojave?

**Comment by mikekazakov** [comment]
I've added a compatibility table to the documentation: https://github.com/mikekazakov/nimble-commander/blob/main/Docs/Help.md#system-requirements.
The last version to support Mojave was `1.2.7`.

**Comment by MrPoetry** [comment]
Thank you. Installed and purchased.

One question:

Is there *any* way to prevent the Backspace key from navigating to the 
enclosing folder?

Cheers,
Mark

**Reply by mikekazakov** [reply]
I'm afraid in that version the behavior of the Backspace key was hardcoded and cannot be changed.
This was fixed in v1.3.0 - since then this shortcut can be adjusted as any others.
